rpm/mir.spec

```diff
@@ -0,0 +1,608 @@
+# Force out of source build
+%undefine __cmake_in_source_build
+
+%ifnarch ppc64
+# Enable LTO on non-ppc64 (c.f. rhbz#1515934)
+%bcond lto 1
+%endif
+
+# Disable ctest run by default
+# They take a long time and are generally broken in the build environment
+%bcond run_tests 0
+
+# Track various library soversions
+%global miral_sover 7
+%global mircommon_sover 11
+%global mircore_sover 2
+%global miroil_sover 7
+%global mirplatform_sover 31
+%global mirserver_sover 64
+%global mirwayland_sover 5
+%global mirplatformgraphics_sover 23
+%global mirplatforminput_sover 10
+
+Name:           mir
+Version:        2.21.1
+Release:        0%{?dist}
+Summary:        Next generation Wayland display server toolkit
+
+# mircommon is LGPL-2.1-only/LGPL-3.0-only, everything else is GPL-2.0-only/GPL-3.0-only
+License:        (GPL-2.0-only or GPL-3.0-only) and (LGPL-2.1-only or LGPL-3.0-only)
+URL:            https://mir-server.io/
+Source0:        https://github.com/canonical/%{name}/releases/download/v%{version}/%{name}-%{version}.tar.xz
+
+BuildRequires:  git-core
```

Comment:
Does it?

Reply:
It's used by `%autosetup -S git_am`, but I can drop this by dropping `-S git_am`.